### PR TITLE
AzureStorage class to work with azure-storage 0.20.0

### PR DESCRIPTION
**Pull request by falcon1kr**
seems there's been breaking changes delivered with 0.20.0

### CODE REVIEW

**Comment by jschneier** [comment]
Hey so just yesterday I merged some fixes into azure storage, can you rebase and make sure your changes still work with that. I'm not 100% sure what versions of Azure is still supported, will have to look at that.

File: storages/backends/azure_storage.py [hunk 1]
```diff
@@ -23,6 +24,8 @@ class AzureStorage(Storage):
     account_name = setting("AZURE_ACCOUNT_NAME")
     account_key = setting("AZURE_ACCOUNT_KEY")
     azure_container = setting("AZURE_CONTAINER")
+    azure_ssl = setting("AZURE_SSL")
+    azure_protocol = 'https:' if azure_ssl else 'http:' if azure_ssl is not None else ''
```
Comment (jschneier) [inline storages/backends/azure_storage.py]: This is hard to follow. How about making it a computed `@property`?

**Comment by falcon1kr** [comment]
just merged in the latest changes that were applied.

**Comment by jschneier** [comment]
hey so since this required another rebase i rebased and squashed down in #86, does that look okay?

**Comment by jschneier** [comment]
merged in #86
